specifyweb/frontend/js_src/lib/components/FormCells/FormTable.tsx

```diff
@@ -82,7 +86,12 @@ export function FormTable<SCHEMA extends AnySchema>({
   readonly onClose: () => void;
   readonly sortField: SubViewSortField | undefined;
   readonly onFetchMore: (() => Promise<void>) | undefined;
+  readonly isCollapsed: boolean | undefined;
+  readonly parentResource?: SpecifyResource<AnySchema> | undefined;
 }): JSX.Element {
+  const [isCollapsed, _, handleCollapsed, handleToggle] =
```

Comment:
I would recommend not duplicating this state here and in IntegratedRecordSelector - duplicating state leads to bugs.
Instead, just have this component receive isCollapsed and handleToggle callbacks
Same for record selector